internal/liqonet/tunnel-operator/labelerOperator.go

```diff
@@ -0,0 +1,91 @@
+package tunneloperator
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/klog/v2"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	liqoutils "github.com/liqotech/liqo/pkg/liqonet/utils"
+)
+
+var (
+	// This labels are the ones set during the deployment of liqo using the helm chart.
+	// Any change to those labels on the helm chart has also to be reflected here.
+	podInstanceLabelKey       = "app.kubernetes.io/instance"
+	podInstanceLabelValue     = "liqo-gateway"
```

Comment:
The instance label values is not guaranteed to be this one if a different release name is specified.